rustls/src/server/tls13.rs

```diff
@@ -504,6 +586,57 @@ mod client_hello {
         common.send_msg(m, false);
     }
 
+    fn decide_if_early_data_allowed(
+        cx: &mut ServerContext<'_>,
+        config: &ServerConfig,
+        client_hello: &ClientHelloPayload,
+        suite: &'static Tls13CipherSuite,
+        resumedata: Option<&persist::ServerSessionValue>,
+    ) -> EarlyData {
+        let early_data_requested = client_hello.early_data_extension_offered();
+        let rejected_or_disabled = if early_data_requested {
+            EarlyData::RequestedButRejected
+        } else {
+            EarlyData::Disabled
+        };
+
+        if let Some(resume) = resumedata {
```

Comment:
Maybe do a `let resume = match resumedata { };` here to reduce rightward drift, below?